cli/app.go

```diff
@@ -21,6 +21,9 @@ If a name is not provided, a random name will be generated.
 If run from a git repository, a 'flynn' remote will be created or replaced that
 allows deploying the application via git.
 
+Options:
+	-r, --remote <remote>  Name of git remote on local repo.
```

Comment:
```text
Name of the git remote to create, empty string for none.
```

Also, it should have `[default: flynn]` at the end so that we don't need a special case below.